lib/discover/file.js

```diff
@@ -0,0 +1,20 @@
+var fs = require('fs');
+
+module.exports.discover = function fileDiscover(urlObj, original, callback) {
```

Comment:
Is there a particular reason why you chose to export `discover`?

This could be as follows:

```
module.exports = function fileDiscover(...) {
    ...
};
```